README.md

```diff
@@ -368,6 +381,36 @@ c2patool sample/C.jpg trust \
 
 **Note:** This sample image should show a `signingCredential.untrusted` validation status since the test signing certificate used to sign them is not contained on the trust lists above.
 
+## Adding a manifest to fragmented BMFF content
+
+The ISO base media file format (BMFF) is a container file format that defines a structure for files that contain time-based multimedia data such as video and audio.
+
+Add a manifest to a fragmented BMFF file by using the `fragment` subcommand, as follows:
+
+```
+c2patool <PATH | PATTERN> fragment [--fragments_glob]
+```
+
+Where `<PATTERN>` is a [glob pattern](https://en.wikipedia.org/wiki/Glob_(programming)).
+
+For example, to add manifest to a video file:
+
+```
+c2patool -m test2.json -o  /1080p_out \
+  /Downloads/1080p/video/avc1/init.mp4 \ 
+  fragment --fragments_glob "seg-*[0-9].m4s"
+```
+
+Or to verify a manifest and fragments:
+```
+c2patool  /Downloads/1080p_out/avc1/init.mp4 \
+  fragment --fragments_glob "seg-*[0-9].m4s"
+```
+
+### Additional option
+
+The `--fragments_glob` option is only available with the `fragment` subcommand and specifies the glob pattern to find the fragments of the asset. The path is automatically set to be the same as the "init" segment so the pattern  only matches segment file names.
```

Comment:
We have an extra space here so we can take the opportunity to make a subtle change:

"segment so the pattern should only matches segment file names, not full paths."

Reply:
"should only matches" is not grammatically correct.   The correct word here is "must" anyway, because we're specifying the requirements for the pattern: "should" is used to make suggestions or recommendations, while "must" is used to express a necessity.

>  ...so the pattern must match only segment file names.

OK?

Reply:
👍